sonic_platform_base/chassis_base.py

```diff
@@ -118,6 +118,26 @@ def get_reboot_cause(self):
         """
         raise NotImplementedError
 
+    def get_controlcard_slot(self):
+        """
+        Retrieves the slot of the control card of the modular chassis
+
+        Returns:
+            An integer, the slot identifier of the control card of the
+            modular-chassis.
+        """
+        return 0
```

Comment:
why the return value is 0?

Reply:
Added back raise NotImplemented. Will catch exception and handle it when used.